### PR TITLE
Change download_url default value

### PR DESCRIPTION
**Pull request by spacedog**
The latest version (7.13.1 for today) isn't available in the location currently
used in module. Use new location as per [1]

[1]: https://www.atlassian.com/software/jira/download

Signed-off-by: Anton Baranov <abaranov@linuxfoundation.org>

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hi @spacedog, can you please have a look at the failed travis jobs?

**Comment by AurelioSilver** [comment]
In the commit message you say the new download URL is: https://www.atlassian.com/software/jira/download/

Which is correct, but in the actual commit you change it to: 
https://product-downloads.atlassian.com/software/jira/downloads/
?

File: manifests/init.pp [hunk 1]
```diff
@@ -90,7 +90,7 @@
   $java_opts                                                        = '',
   $catalina_opts                                                    = '',
   # Misc Settings
-  Variant[Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $download_url          = 'https://downloads.atlassian.com/software/jira/downloads/',
+  Variant[Stdlib::HTTPUrl, Stdlib::HTTPSUrl] $download_url          = 'https://product-downloads.atlassian.com/software/jira/downloads/',
```
Comment (alexjfisher) [inline manifests/init.pp]: I think we need to lose the trailing slash or looking at the travis failures we get a `403`.

Comment (nambrosch) [reply manifests/init.pp]: yes it causes double-slash -

`Error: Execution of '/usr/bin/curl https://downloads.atlassian.com/software/jira/downloads//atlassian-jira-software-7.13.1.tar.gz -o /tmp/atlassian-jira-software-7.13.1.tar.gz_20190223-8941-1h09xbl -fsSLg --max-redirs 5 --proxy ...' returned 22: curl: (22) The requested URL returned error: 404 Not Found`

Comment (alexjfisher) [reply manifests/init.pp]: Odd that you got a `404` when the acceptance tests got a `403`
https://travis-ci.org/voxpupuli/puppet-jira/jobs/488570611#L1211
> Error: /Stage[main]/Jira::Install/Archive[/tmp/atlassian-jira-6.3.4a.tar.gz]/ensure: change from 'absent' to 'present' failed: Execution of '/opt/puppetlabs/puppet/bin/curl https://product-downloads.atlassian.com/software/jira/downloads//atlassian-jira-6.3.4a.tar.gz -o /tmp/atlassian-jira-6.3.4a.tar.gz_20190204-2261-jmy6j0 -fsSLg --max-redirs 5' returned 22: curl: (22) The requested URL returned error: 403 Forbidden

Either way, I think the double slashes are the problem.

**Pull request closed without merging**